prow/cmd/deck/static/prow/prow.ts

```diff
@@ -649,42 +653,31 @@ function redraw(fz: FuzzySearch, pushState: boolean = true): void {
                 if (pulls.length) {
                     r.appendChild(cell.prRevision(`${org}/${repo}`, pulls[0]));
                 } else {
-                    r.appendChild(cell.text(""));
+                    r.appendChild(cell.text("-"));
                 }
             } else if (type === "batch") {
                 r.appendChild(batchRevisionCell(build));
             } else if (type === "postsubmit") {
                 r.appendChild(cell.commitRevision(`${org}/${repo}`, base_ref, base_sha, base_link));
             } else if (type === "periodic") {
-                r.appendChild(cell.text(""));
+                r.appendChild(cell.text("-"));
             }
         } else {
             // Don't render identical cells for the same PR/commit.
-            r.appendChild(cell.text(""));
-            r.appendChild(cell.text(""));
-        }
-        if (spyglass) {
```

Comment:
why deleting this?

Reply:
The spyglass icon is redundant with the link to Results. This may cause confusion to users. In addition, removing it will also give us more page real estate to add header descriptors.

Reply:
This has been added back, due to potential issues it may cause

Reply:
The spyglass icon will always redirect to the spyglass, however, the Job link next to it may or may not redirect to spyglass. I think leaving the icon as is would be the best option.